### PR TITLE
ascanrulesBeta: Add policy alert tags

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Add note and fix note.
- Scan rules > Add PolicyTag values.
- Unit tests > Assert the new tags.
- Build file > Update the required version of commonlib.

## Related Issues
- zaproxy/zap-extensions#5930

## Checklist
- [na] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureScanRule.java [hunk 1]
```diff
@@ -47,12 +49,6 @@
 public class BackupFileDisclosureScanRule extends AbstractAppPlugin
         implements CommonActiveScanRuleInfo {
 
-    private static final Map<String, String> ALERT_TAGS =
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureScanRule.java:50]: This should still be done during initialisation.

File: addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/HttpOnlySiteScanRuleUnitTest.java [hunk 2]
```diff
@@ -61,25 +62,27 @@ void shouldReturnExpectedMappings() {
         // Then
         assertThat(cwe, is(equalTo(311)));
         assertThat(wasc, is(equalTo(4)));
-        assertThat(tags.size(), is(equalTo(3)));
+        assertThat(tags.size(), is(equalTo(4)));
+        System.out.println(tags.keySet());
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/HttpOnlySiteScanRuleUnitTest.java:66]: Remove.

File: addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/Text4ShellScanRuleUnitTest.java [hunk 3]
```diff
@@ -119,7 +121,20 @@ void shouldHaveExpectedNumberOfAlertTags() {
         // Given / When
         Map<String, String> alertTags = rule.getAlertTags();
         // Then
-        assertThat(alertTags.size(), is(equalTo(5)));
+        assertThat(alertTags.size(), is(equalTo(8)));
+    }
+
+    @Test
+    void shouldHaveExpectedPolicyAlertTags() {
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/Text4ShellScanRuleUnitTest.java:128]: We should refactor all the alert tags/policies/mappings verification so they are done consistently throughout the codebase, we have now at least 4 ways (if not more).

**Comment by kingthorin** [comment]
@thx202 I still have more to work through but is that more what you had in mind? (Static blocks and using the more ummm verbose unit test?)

**Comment by thc202** [comment]
Yes, but I'd keep them immutable. I didn't say a more verbose test.

**Comment by thc202** [comment]
If you mean one of `shouldReturnExpectedMappings`, yes, that looks good.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**